apps/dotcom/client/src/utils/clipboard.ts

```diff
@@ -0,0 +1,14 @@
+/** @internal */
+export async function writeToClipboard(result: Promise<Blob | ''>) {
```

Comment:
wait, i know this isn't your code and you're refactoring but why does one path `await result` and the other doesn't? shouldn't they both need to `await`?

Reply:
I think that in async contexts you need to pass a promise to `clipboard.write` to make Safari happy.
https://wolfgangrittner.dev/how-to-use-clipboard-api-in-firefox/

Reply:
gotcha, ok makes sense. if you get a chance - add your comment in the code!